### ISSUE TITLE
Add friends?

### ISSUE THREAD

**Issue by Thyrus1** [Kaeios/Nations#10]
Is there any chance that a friends section can be added? where you do like /n friend add or something like that

**Comment by Zoneman1972** [comment]
Yes, I would also like to see something like this added, if its possible.
Allowing a Nation to set another Nation as "Friendly" status, which in turn allows the members of that Nation the ability to build/interact in each others claims.
